src/marqo/core/unstructured_vespa_index/unstructured_vespa_schema.py

```diff
@@ -219,6 +219,7 @@ def _generate_unstructured_schema(cls, marqo_index: UnstructuredMarqoIndex) -> s
                     summary {cls._LONGS_STRINGS_FIELDS} type map<string, string> {{}}
                     summary {cls._SHORT_STRINGS_FIELDS} type map<string, string> {{}}
                     summary {cls._STRING_ARRAY} type array<string> {{}}
+                    summary {cls._BOOL_FIELDS} type map<string, byte> {{}}
```

Comment:
Looks like tests have been missing this bug. We need to add tests to cover this now

Reply:
resolved